### PR TITLE
UML-1248 additional activation key information

### PR DESCRIPTION
**Pull request by cooperaj**
# Purpose

Insert information page about the addition of older lpas by requesting an activation key

Fixes UML-1248

## Checklist

* [x] I have performed a self-review of my own code
* [ ] ~I have added relevant logging with appropriate levels to my code~
* [ ] ~I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant~
* [ ] ~I have added tests to prove my work~
* [x] I have added welsh translation tags and updated translation files
* [ ] I have run an accessibility tool on any pages I have made changes to and fixed any issues found
* [ ] The product team have tested these changes

### CODE REVIEW

File: service-front/app/src/Actor/templates/actor/request-activation-key-info.html.twig [hunk 1]
```diff
@@ -0,0 +1,76 @@
+{% extends '@actor/layout.html.twig' %}
+
+{% block html_title %}{% trans %}Ask for an activation key{% endtrans %} - {{ parent() }}{% endblock %}
```
Comment (GemTay) [inline service-front/app/src/Actor/templates/actor/request-activation-key-info.html.twig:3]: This title will need to differ from the form page title so they can be tracked separately in google analytics

File: service-front/app/src/Actor/src/Handler/RequestActivationKeyInfoHandler.php [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+/**
+ * Class RequestActivationKeyInfoHandler
+ * @package Actor\Handler
+ * @codeCoverageIgnore
+ */
+class RequestActivationKeyInfoHandler extends AbstractHandler implements UserAware
+{
+    use User;
+
+    private RequestActivationKey $form;
```
Comment (GemTay) [inline service-front/app/src/Actor/src/Handler/RequestActivationKeyInfoHandler.php:26]: I don't think this form is needed here, unless i'm missing something?

Comment (cooperaj) [reply service-front/app/src/Actor/src/Handler/RequestActivationKeyInfoHandler.php:26]: Quite right. Thought I'd got rid of all the old stuff. Will sort now.

File: service-front/app/src/Actor/templates/actor/request-activation-key-info.html.twig (hunk 1 above)
Comment (cooperaj) [reply service-front/app/src/Actor/templates/actor/request-activation-key-info.html.twig:3]: True. Will see what Jude/Uchenna suggest.

**Review by GemTay — APPROVED** [review]
LGTM 👍

**Review by GemTay — APPROVED** [review]

**Pull request merged**